### DISCUSSION TITLE
Metrics Persistence

### DISCUSSION THREAD

**Discussion by jvxis** [onemorebsmith/kaspa-stratum-bridge#52] [Q&A] [answered]
Hi @onemorebsmith, I'd like to know that if I stop the stratum service, I will lose all metrics and, consequently, all history.

Thanks JVX

**Comment by onemorebsmith** [comment, marked as answer]
Answered on discord but yes, it will persist as long as you don't delete the docker volume for prom

**Reply by jvxis** [reply]
Yes, thanks. I just keep it registered here.
